contracts/rbn-staking/FeeCustody.sol

```diff
@@ -110,23 +114,22 @@ contract FeeCustody is Ownable {
       IWETH(address(distributionToken)).deposit{value: address(this).balance}();
     }
 
-    for (uint256 i; i < lastAssetIdx; i++) {
+    for (uint256 i; i < assets.length; i++) {
```

Comment:
Nit, caching the `assets.length` in a variable before the loop saves a lot of gas

```solidity
uint len = assets.length
for (uint256 i; i < len; i++) {
```